### ISSUE TITLE
There is no function in slot after swapping slot with production which has a repository connected

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-azurefunctions#3064]
**OS:** Win 10
**Build Version:** 20220228.3

**Repro Steps:**
1. Create a Linux Node.js Function App.
2. Deploy a JavaScript/TypeScript project to this Function App.
3. Expand the Function App -> Deployment node -> Connect a repository to it.
4. Create a slot for this Function App.
5. Execute "Swap Slot" action with production.
6. Check whether there is a function in slot or not.

**Expect:**
There is a function in slot.

**Actual:**
There is no function in slot.
![image](https://user-images.githubusercontent.com/45614674/156345319-f120fbf1-9311-4ce0-9cac-075fad0aa3e1.png)

**More Info:**
This issue also reproduces for Python Functions App.

**Comment by nturinski** [comment]
Does anything else get swapped like the Application Settings or Files?  I'm wondering if maybe the entire swap did nothing or if just the function triggers aren't swapping.

**Comment by v-ruizh** [comment]
@nturinski Both the Application Settings and Files are swapped.

**Comment by wwlorey** [comment]
@v-ruizh we're thinking this is a timing issue. Can you try reproing again but check the Functions node after waiting for a few minutes?

**Comment by v-ruizh** [comment]
@wwlorey This issue also reproduces after waiting for a few hours. Refresh can't fix this issue.

**Comment by nturinski** [comment]
@v-ruizh I just noticed, but in the screenshot, there are no functions in the production app either.  Is the function app being deployed correctly?

**Comment by v-ruizh** [comment]
@nturinski Yes, the function app being deployed correctly. Function exists in the production app before swapping slot.
